src/cdatrie.c

```diff
@@ -1,4 +1,4 @@
-/* Generated by Cython 0.29.11 */
+/* Generated by Cython 0.29.14 */
```

Comment:
We should remove cython-generated files and just regenerate them on each build.

Reply:
I agree, [I mentioned the same above](https://github.com/pytries/datrie/pull/76#issuecomment-574286987).

Reply:
I did not remove it, but I added cythonize support in #78